parsers/wiki_parser.py

```diff
@@ -165,12 +160,8 @@ def create_responses_text_and_link_list(responses_source):
 
 def parse_response(text):
     # Special cases
-    if '(broken file)' in text:
-        return ''
-    if 'versus (TI ' in text:
-        return ''
-    if 'Ceeeb' in text:
-        return ''
+    if any(excluded_case in text for excluded_case in ['(broken file)', 'versus (TI ', 'Ceeeb']):
```

Comment:
Isn't

    if text in ['(broken file)', 'versus (TI ', 'Ceeeb']:
        return None

enough?

Reply:
Not really. Some files have broken audio files, and they have the `(broken file)` text in them, e.g. `Alliance (broken file)`. There's plenty more [here](https://dota2.gamepedia.com/Announcer_responses) too. I'm checking if `(broken file)` is in the text, not the other way around.

Reply:
Yeah, you're right, sorry.